packages/react-sdk/src/core/components/Video/BaseVideo.tsx

```diff
@@ -1,38 +1,36 @@
 import {
-  DetailedHTMLProps,
+  ComponentPropsWithRef,
   ForwardedRef,
   forwardRef,
   useEffect,
   useState,
-  VideoHTMLAttributes,
 } from 'react';
-import clsx from 'clsx';
 import { Browsers } from '@stream-io/video-client';
 
-export type VideoProps = DetailedHTMLProps<
-  VideoHTMLAttributes<HTMLVideoElement>,
-  HTMLVideoElement
-> & {
+export const applyElementRef = <T extends HTMLElement | null>(
```

Comment:
👍

Reply:
Ah, I should probably move this to utilities.